src/components/Main/Main.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import { useCountries } from '../../hooks/countries.js';
+import CountryCard from '../CountryCard/CountryCard.js';
+import Interface from '../Interface/Interface.js';
+
+export default function Main() {
+  const [continentFilter, setContinentFilter] = useState('');
+
+  return (
+    <main>
+      <Interface setContinentFilter={setContinentFilter} />
+      {useCountries()
```

Comment:
Interesting way to use the custom hook but won't work if you end up returning multiple variables.